shopify_auth/session_tokens/views.py

```diff
@@ -18,10 +28,18 @@ def get_scope_permission(request, myshopify_domain):
         getattr(settings, "SHOPIFY_APP_API_VERSION", "unstable"),
     ).create_permission_url(settings.SHOPIFY_APP_API_SCOPE, redirect_uri)
 
+    embedded = request.GET.get("embedded")
+    if not embedded or embedded != "1":
+        return HttpResponseRedirect(permission_url)
+
+    host = request.GET.get("host")
+    if not host:
+        return HttpResponse("expected host query parameter when embedded=1 is present")
```

Comment:
Wouldn't be more systematic to raise an exception instead?

I believe it would also be better to increase chance that monitoring systems like Sentry will help to notify developers about such situation.

Reply:
makes sense, thanks